projects/aws-containers/hello-eks-anywhere/Help.mk

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Is this auto generated spacing? Or can the extra spacing be removed?

Reply:
That is a generated file